### PR TITLE
dotCMS/core#26853 Handle Errors from endpoint with AI Plugin

### LINKED ISSUE #26853: [UI] Implement Handle Errors from endpoint
### Parent Issue
#26495 

### User Story
As a user, I wish to visually see when the image generation process fails, enabling me to quickly understand and respond to any issues that occur during this process.

### Acceptance Criteria
- Implement a clear visual indicator in the UI for when the image generation fails.
- Implement the design from UI in Figma
- Users should be able to understand the nature of the error from the visual indicator.

### dotCMS Version
master

### Proposed Objective
User Experience

### Proposed Priority
Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.
Screenshot 2023-11-30 at 11 34 55 AM

### Assumptions & Initiation Needs
- Assumes that the current system does not have sufficient visual indicators for image generation errors.
- Requires access to the UI design and development resources to implement the visual indicators.

### PR DESCRIPTION
**Pull request by oidacra**
### Proposed Changes
* Handle errors using AI Plugins

Block Editor as App (JSP)

https://github.com/dotCMS/core/assets/1909643/05957cbf-fafb-417f-a9a5-862454ee2110

Block Editor as Component (editing Sidebar at Edit Page)

https://github.com/dotCMS/core/assets/1909643/18abe3aa-2309-4688-8e05-fdc53799c910

### CODE REVIEW

**Review by fmontes — APPROVED** [review]

File: core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.html [hunk 1]
```diff
@@ -27,3 +27,10 @@
         </span>
     </form>
 </ng-container>
+<p-confirmDialog
+    [acceptIcon]="null"
+    [rejectIcon]="null"
```
Comment (zJaaal) [inline core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.html:32]: this doesn't work in the new version we have of primeNG. Now it's `hidden`

File: core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/ai-image-prompt.component.html [hunk 2]
```diff
@@ -29,3 +29,12 @@
         </div>
     </p-dialog>
 </ng-container>
+
+<p-confirmDialog
+    [acceptIcon]="null"
+    [rejectIcon]="null"
```
Comment (zJaaal) [inline core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/ai-image-prompt.component.html:35]: Same here

**Review by zJaaal — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-block-editor/project.json [hunk 3]
```diff
@@ -44,6 +45,11 @@
                 "namedChunks": true
             },
             "configurations": {
+                "localhost": {
+                    "sourceMap": true,
+                    "optimization": false,
+                    "watch": true
+                },
```
[the remark this answers is not in the document]
Comment (oidacra) [reply core-web/apps/dotcms-block-editor/project.json:52]: Yes. I need a build Block-Editor with the maps to check the errors.

File: core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.html (hunk 1 above)
Comment (oidacra) [reply core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.html:32]: yes, you are right. 
But in the end I did
```
  rejectVisible: false,
  acceptVisible: false
```
direct at the service.

File: core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/ai-image-prompt.component.html (hunk 2 above)
Comment (oidacra) [reply core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/ai-image-prompt.component.html:35]: deleted

**Review by zJaaal — APPROVED** [review]

**Pull request merged**